app/src/modules/settings/routes/extensions/components/extension-item-options.vue

```diff
@@ -23,7 +33,7 @@ const { t } = useI18n();
 						<v-icon name="mode_off_on" />
 					</v-list-item-icon>
 					<v-list-item-content>
-						{{ enabled ? t('disable') : t('enable') }}
+						{{ status }}
```

Comment:
Would it be possible to show both "enable all" and "disable all" when the bundle is "partially enabled"? That sounds like it would be better UX than disabling them all first and then re-enabling them 🤔

Reply:
Done, I have added icon color as well for easier differentiation

#### UI Samples
<img width="383" alt="Screenshot 2024-01-04 at 4 19 55 PM" src="https://github.com/directus/directus/assets/44623501/40a17a19-5c5c-46f7-bf74-ebb651ac262a">
<img width="560" alt="Screenshot 2024-01-04 at 4 24 26 PM" src="https://github.com/directus/directus/assets/44623501/9407399c-894e-4e25-9282-08db1c08f20a">
<img width="560" alt="Screenshot 2024-01-04 at 4 24 31 PM" src="https://github.com/directus/directus/assets/44623501/dd55db13-a240-4576-9ff6-a04e7002308d">